packages/schema/src/config/common.ts

```diff
@@ -309,10 +313,7 @@ export default defineUntypedSchema({
    */
   alias: {
     $resolve: async (val, get) => {
-      const srcDir = await get('srcDir')
-      const rootDir = await get('rootDir')
-      const dirAssets = await get('dir.assets')
-      const dirPublic = await get('dir.public')
+      const [srcDir, rootDir, dirAssets, dirPublic] = await Promise.all(get('srcDir'), get('rootDir'), get('dir.assets'), get('dir.public'))
```

Comment:
Code style: usually variables names are like `assetsDir` and `publicDir` in reverse

Reply:
Thanks!